package/MDAnalysis/converters/OpenMM.py

```diff
@@ -85,7 +85,7 @@ def _format_hint(thing):
         """Can this reader read *thing*?
         """
         try:
-            from simtk.openmm.app import Simulation
```

Comment:
Ideally we would like to support older versions which require the simtk import. Could you re-arrange this to account for that?

Reply:
In practice, it would be something in the line of:

* try importing directly from `openmm`
* if it fails, fall back to importing from `simtk`
* if that fails too, we do not have openmm

Reply:
Thanks a lot, I have pushed another commit to keep compatibility, PTAL